src/tito/tagger/main.py

```diff
@@ -132,10 +133,42 @@ def _tag_release(self):
         new_version = self._bump_version()
         self._check_tag_does_not_exist(self._get_new_tag(new_version))
         self._update_changelog(new_version)
+        # Update pkg managers used by the project (e.g. Cargo, Cabal)
+        # XXX: merge setup_py and pom_xml into this method ?
+        self._update_pkg_managers(new_version)
         self._update_setup_py(new_version)
         self._update_pom_xml(new_version)
         self._update_package_metadata(new_version)
```

Comment:
Not specific to this PR, but I'm thinking it might be time to rethink this portion of the VersionTagger.  Right now we're updating specific files: setup.py, pom.xml, etc. Would be better if we had some sort of registry so this code would change to :+1: 

```
self._update_SOME_GOOD_NAME(new_version)
```

That method could look through a registry which was configured in the tito.props.

```
def _update_SOME_GOOD_NAME(version):
   for file in self.registry:
      DO THE RIGHT THING :)
```